exonum-java-binding/core/rust/ejb-app/start_cryptocurrency_node.sh

```diff
@@ -38,6 +38,7 @@ echo "PROJ_ROOT=${EJB_ROOT}"
 header "PREPARE PATHS"
 
 CORE_TXT="core/target/ejb-core-classpath.txt"
+# FIXME: Remove the CRYPTOCURRENCY_TXT as it is no longer generated/needed, see PR for ECR-3005
```

Comment:
When should it be removed?

Reply:
When configuration is stabilized in the target feature branch. @skletsun knows :-)